junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -1057,6 +1057,41 @@ public static boolean isMethodPresent(Class<?> clazz, Predicate<Method> predicat
 		// @formatter:on
 	}
 
+	/**
+	 * Get the values of the specified fields on the given instance.
+	 *
+	 * @param fields list of fields on a class; never {@code null}
+	 * @param object instance of a class; never {@code null}
+	 * @return an immutable list of values of the specified fields on the given instance
+	 */
+	public static List<Object> getFieldValuesForInstance(List<Field> fields, Object object) {
+		return getFieldValuesForInstance(fields, object, Object.class);
+	}
+
+	/**
+	 * Get the values of the specified fields of specified type on the given instance.
+	 *
+	 * @param fields list of fields on a class; never {@code null}
+	 * @param object instance of a class; never {@code null}
+	 * @param fieldType the type of all fields from fields; never {@code null}
+	 * @return an immutable list of values of the specified fields on the given instance
+	 */
+	@SuppressWarnings("unchecked")
+	public static <T> List<T> getFieldValuesForInstance(List<Field> fields, Object object, Class<T> fieldType) {
+		Preconditions.notNull(fields, "Fields must not be null");
+		Preconditions.notNull(object, "Object must not be null");
+		Preconditions.notNull(fieldType, "Field type must not be null");
+
+		return fields.stream().map(f -> {
```

Comment:
Please rewrite this to make use of `org.junit.platform.commons.util.ReflectionUtils.makeAccessible(T)`.

Reply:
makeAccessible will be called for all fields